tools/ci-check.sh

```diff
@@ -5,6 +5,6 @@ set -e
 yarn build
 yarn lint
 yarn test:unit -- -b ChromeHeadless_Travis -b Firefox
-yarn test:styles
+yarn test:build
```

Comment:
Did we mean to drop `test:styles` in this diff? Do we not need it anymore?

Reply:
@joshblack Good question, `test:styles` is renamed to `test:build`, given our Jest test contains stuffs more than styles. Meaning the styles tests are _not_ gone.